util/systemd/rustic-backup@.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=rustic --use-profile %I backup
+
+[Service]
+Nice=19
+IOSchedulingClass=idle
+KillSignal=SIGINT
+ExecStart=/usr/bin/rustic --use-profile %I backup
```

Comment:
What about adding a comment that this needs a config file `%I.toml` in`/etc/rustic/` or the users config dir `~/.config/rustic/`?

Reply:
I'm not sure... Why put the comment here, shouldn't it be "general knowledge" that users have to have config files in order to use profiles?

Reply:
I thought more about the fact that people directly see the connection between a `rustic-backup@my-profile.service` and rustic using the profile `my-profile`.

Reply:
It's kind of in the description already

```
Description=rustic --use-profile %I backup
```

And I think I'd rather put it in the documentation somewhere... The units need to be documented anyways, otherwise nobody will find them in the first place.